internal/eval/evalers.go

```diff
@@ -1194,10 +1276,14 @@ func newExtensionEval(name types.Path, args []Evaler) Evaler {
 			return newErrorEval(fmt.Errorf("%w: %s takes %d parameter(s)", errArity, name, i.Args))
 		}
 		switch {
-		case name == "ip":
-			return newIPLiteralEval(args[0])
+		case name == "datetime":
```

Comment:
Optional: but it might be nice if all these strings were constants in the ExtMap package.  Something like ... 
```
const PathIP = "ip" 
```

Then the map could use those references as well.